lib/services.js

```diff
@@ -478,6 +479,7 @@ export default {
         const mpuBucketName = `${constants.mpuBucketPrefix}${bucketName}`;
         metadata.getBucket(mpuBucketName, log, (err, mpuBucket) => {
             if (err) {
+                log.error('error from metadata', { error: err });
```

Comment:
Back in the days we reasoned this as  - since we don't have enough error context in lower layers warn/trace/debug were chosen (this needs to be unified) and then we would log error in the api layer by filtering out the error codes which we think are safe.
If we just check for 404 there may be cases where 404 is not even expected like for example, putobjectMD call.